### PR TITLE
Update Submission.md

### PR DESCRIPTION
**Pull request by Samuel7050**
Name - Ojo Samuel
I developed a serverless API using AWS Lambda, DynamoDB, and API Gateway, with CloudWatch Logs enabled for monitoring the Lambda function. To automate the updating of resume data in DynamoDB, I set up a CI/CD pipeline with GitHub Actions and managed all infrastructure as code using Terraform. This approach eliminates the need for manual configuration in the AWS console, ensuring seamless updates to resume data whenever changes are pushed to the repository and enhancing code reusability. Below is the link to the technical article 

https://dev.to/gbenga700/serverless-resume-api-with-terraform-and-github-actions-4b6a

Thanks for the engaging challenge. This has helped me in developing hands on experience in using AWS services.

### CODE REVIEW

**Comment by rishabkumar7** [comment]
Hi @Samuel7050 I would suggest adding some info and detail in the PR.
I have looked at the API URL and git, looks good.

**Comment by Samuel7050** [comment]
Name - Ojo Samuel
I developed a serverless API using AWS Lambda, DynamoDB, and API Gateway, with CloudWatch Logs enabled for monitoring the Lambda function. To automate the updating of resume data in DynamoDB, I set up a CI/CD pipeline with GitHub Actions and managed all infrastructure as code using Terraform. This approach eliminates the need for manual configuration in the AWS console, ensuring seamless updates to resume data whenever changes are pushed to the repository and enhancing code reusability.

Thanks for the engaging challenge. This has helped me in developing hands on experience in using AWS services.

File: docs/submissions.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ This is the place to add your own Cloud Resume API project!
 | Hein Htet Win | https://iz15tg1yjg.execute-api.ap-southeast-1.amazonaws.com/Prod/ | https://github.com/heinhtetwin/cloud-resume-challenge
 | Suryendu Bhattacharyya | https://ozdzf07h9c.execute-api.eu-central-1.amazonaws.com/prod/resume | https://github.com/SuryenduB/Cloud_Resume_JSON
```
Comment (rishabkumar7) [inline docs/submissions.md:16]: Can you remove the whitespace.

Comment (Samuel7050) [reply docs/submissions.md]: Done

**Comment by Samuel7050** [comment]
Thanks @rishabkumar7 
Just saw the give away video and i Won . 
See below my socials
www.linkedin.com/in/samuel-ojo-oluwagbenga
Many Thanks to you and [Ifeanyi Otuonye, MBA

**Pull request merged**